flang/test/Parser/OpenMP/scan.f90

```diff
@@ -0,0 +1,40 @@
+! RUN: %flang_fc1 -fdebug-unparse -fopenmp %s | FileCheck --ignore-case %s
+! RUN: %flang_fc1 -fdebug-dump-parse-tree -fopenmp %s | FileCheck --check-prefix="PARSE-TREE" %s
+
+! Check for parsing scan directive
+subroutine test_scan_inclusive()
+  implicit none
+  integer, parameter :: n = 100
+  integer a(n), b(n)
+  integer x, k
```

Comment:
Nit: I'd suggest making these subroutine arguments and to remove the initialization. That way we minimize the size of the PFT. It's not a blocker, though, you can leave that unchanged if you prefer.

Reply:
Updated. Thank you!